### PR TITLE
Visualstudio2017

### PR DESCRIPTION
**Pull request by handrok**

### CODE REVIEW

File: libmscore/staff.cpp [hunk 1]
```diff
@@ -414,11 +414,13 @@ Fraction Staff::timeStretch(int tick) const
 TimeSig* Staff::timeSig(int tick) const
       {
       auto i = timesigs.upper_bound(tick);
-      if (i != timesigs.begin())
+	  if (i != timesigs.begin())
```
Comment (Jojo-Schmitz) [inline libmscore/staff.cpp]: whitespace damage, should be 6 spaces like it was before the change

Comment (Jojo-Schmitz) [reply libmscore/staff.cpp]: The tabs in the lines below should get fixed to spaces too

**Comment by Jojo-Schmitz** [comment]
rebasing should have worked too?

**Comment by handrok** [comment]
see https://github.com/musescore/MuseScore/pull/3796 @Jojo-Schmitz Your review has beed fixed in the link

**Comment by Jojo-Schmitz** [comment]
Ah, thanks

**Pull request closed without merging**